frontend/mainwindow.cpp

```diff
@@ -96,16 +94,16 @@ void MainWindow::recentCycle()
     if (ui_->instrumentsBox->currentText() != "")
     {
         if (cycleIndex != -1)
+        {
             ui_->cyclesBox->setCurrentIndex(cycleIndex);
-        else if (ui_->cyclesBox->currentText() != "")
+        }
+        else
+        {
             ui_->cyclesBox->setCurrentIndex(ui_->cyclesBox->count() - 1);
+        }
```

Comment:
Any reason for adding in the braces here?  When you only have a single line of code encapsulated in them I find that it actually makes it less readable.